docs/html/pmc-manual.html

```diff
@@ -225,6 +226,109 @@
 	Address: 0x0	size: 64</pre>
 <p>
       </p>
+<p>
+	Pmemcheck also allows tracking of persistent memory transactions.
+	To learn more about the built-in transaction support in pmemcheck
+	see <a class="xref" href="pmc-manual.html#pmc-manual.transactions" title="14.1.4.�Pmemcheck transaction support">Transactions</a>. The primary type of issue
+	found by pmemcheck are stores made to regions of memory not tracked
+	by transactions.  An example output of the out-of-transaction stores
+	has the following form:
+	</p>
+<pre class="programlisting">
+Number of stores made outside of transactions: 2
+Stores made outside of transactions:
+[0]    at 0x400C3E: main (trans_impl_nest.c:39)
+	Address: 0x4fec000	size: 1
+[1]    at 0x400C58: main (trans_impl_nest.c:42)
+	Address: 0x4fec018	size: 8</pre>
+<p>
+      </p>
+<p>
+	A different type of error message related to the transactions
+	functionality within pmemcheck, are active transactions on program exit.
+	This kind of error shows that some transactions have not finished
+	properly. This can lead to data corruption and possibly performance
+	degradation.  If any transactions are active on exit, a report with the
+	following form is issued:
+	</p>
+<pre class="programlisting">
+Number of active transactions: 1
+[0]    at 0x400AE5: main (trans_impl_nest.c:32)
+	tx_id: 1	 nesting: 1</pre>
+<p>
+      </p>
+</div>
+<div class="sect2">
+<div class="titlepage"><div><div><h3 class="title">
+<a name="pmc-manual.transactions"></a>14.1.4.�Pmemcheck transaction support</h3></div></div></div>
+<p>
+	Contemporary hardware architectures guarantee that only stores up to
+	a certain size can be made atomically. Here atomicity refers to non-torn
+	writes and not concurrency safety. At the time of writing this
+	documentation the x86-64 architecture supports 8 byte atomic writes.
+	This for most cases is insufficient in real world applications.
+	Therefore the need to introduce transactions, which allow larger ranges
+	of persistent memory to be changed atomically. Whether transactions
+	are rolled-back, redone or employ some other technique is of no
+	importance to pmemcheck. What is important is that pmemcheck knows
+	which regions of persistent memory are tracked by the transaction and
+	can be modified freely while the transaction is active.
+      </p>
+<p>
+	To enable transaction support in pmemcheck, use the
+	<a class="xref" href="pmc-manual.html#opt.transactions">--transactions</a> command line option. To inform
+	pmemcheck of a new transaction start, use one of two available macros
+	<a class="xref" href="pmc-manual.html#crm.start_tx">VALGRIND_PMC_START_TX</a> or <a class="xref" href="pmc-manual.html#crm.start_tx_n">VALGRIND_PMC_START_TX_N</a>.
+	Depending on whether your application uses flattened, implicit
```

Comment:
In case of per-thread transactions - does the tool detect the case when a thread attempts to modify memory regions added to TX by another thread?  I know it's more like thread synchronization issue and there other tools to do that, but...